web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -544,6 +554,12 @@ export default defineComponent({
             mdi-refresh
           </v-icon>
         </v-btn>
+        <v-snackbar
+          v-model="snackbar"
+          color="success"
+          timeout="3000">
+            Validation Passed! You can now submit or continue editing.
```

Comment:
Almost there: the `>` on line 560 should be on its own line, at the same indentation as the corresponding `<`, and the text ("Validation Passed!...") is indented one level (2 spaces) too far.

Reply:
This is fixed in the newest commit now